app/models/employer_profile.rb

```diff
@@ -850,6 +850,46 @@ def trigger_notices(event)
     ShopNoticesNotifierJob.perform_later(self.id.to_s, event)
   end
 
+  def upload_document(file_path,file_name,subject,size)
+    #doc_uri = Aws::S3Storage.save(file_path,'id-verification')
+    #file = File.open(file_path, "r:ISO-8859-1")
+
+    tmp_file = "#{Rails.root}/tmp/#{file_name}"
+    id = 0
```

Comment:
Please sync with each other and settle on a correct way to solve this. Sagar is the one who implemented the S3Storage service. Did you speak with him?